e2e/doc/condition_table.md

```diff
@@ -0,0 +1,17 @@
+ ## node_power_off.robot
```

Comment:
Currently in the test case of node powering off, since the similarity of each situation is high. Therefore, I've created a ticket https://github.com/longhorn/longhorn/issues/6532 to find the proper way to describe the case clearly.
<img src="https://github.com/longhorn/longhorn-tests/assets/17548901/2c545a52-2052-4fbc-8ef6-89420d797163" alt="drawing" width="500"/>